posts/models.py

```diff
@@ -1,17 +1,36 @@
 from django.db import models
 from users.models import User
+from multiselectfield import MultiSelectField
 # Create your models here.
 
 
+class Ingredients(models.Model):
+    name = models.CharField(max_length=120, blank=False)
+
+
 class Post(models.Model):
+    owner = models.ForeignKey(User, on_delete=models.CASCADE)
+
+    INGREDIENT_CHOICES = (
+        ('tomato', 'Tomato'),
+        ('eggplant', 'Eggplant'),
+        ('celery', 'Celery'),
+        ('egg', 'Egg'),
+        ('milk', 'Milk'),
+        ('fish', 'Fish'),
+        ('chicken', 'Chicken'),
+        ('oil', 'Oil')
+    )
+    # picked_ingredients
+    ingredients = models.ManyToManyField(Ingredients)
+    choices = MultiSelectField(choices=INGREDIENT_CHOICES, max_length=1000, blank=True)
```

Comment:
You should not add `choices` field. You should list existing `Ingredients` and user select these.
If you list this static list to users, when a new ingredient object is added you should update your code/list.